release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2023,6 +2023,20 @@ As a workaround, use fully qualified names, such as `registry.redhat.io/ubi8/ubi
 +
 Optionally, you can update the image registry settings by link:https://docs.openshift.com/container-platform/4.9/openshift_images/image-configuration.html#images-configuration-shortname_image-configuration[adding registries that allow image short names]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011293[*BZ#2011293*])
 
+* The `oc adm release extract --tools` command fails when an image that is hosted in the local registry is specified. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1823143[BZ#1823143])
+
+* In a Single Node OpenShift (SNO) cluster that uses real-time kernel version 8.4, the time taken to create a pod is two and a half times slower than with the standard kernel version 8.4. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1975356[BZ#1975356])
+
+* In a SNO cluster deployed with 150 or more pods, pod recovery times are slow after a node reboot. The slow recovery is due to the following error: "timed out waiting for OVS port binding". When the issue occurs there are usually mutiple failures. The pods eventually recover. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986216[BZ#1986216])
+
+* There is an instability issue with the inbox iavf driver that is evident when a DPDK based workload is deployed in {product-title} version 4.8.5. The issue is also apparent when a DPDK workload is deployed on a host running RHEL for Real Time 8. The issue occurs in hosts with Intel XXV710 NICs installed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2000180[BZ#2000180])
+
+* An error occurs during SNO cluster provisioning where Bootkube tries to use `oc` towards the end of the cluster bootstrap process. The Kube API receives a shutdown request and this causes the cluster bootstrap process to fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2010665[BZ#2010665])
+
+* A clock jump error occurs in the `linuxptp` subsystem that is deployed by the PTP Operator. The reported error is: "clock jumped backward or running slower than expected!". The error is encountered in a host with an Intel E810 NIC installed in a {product-title} version 4.8 cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2013478[BZ#2013478])
+
+* Deploying an {product-title} version 4.9 SNO cluster after a successful 4.8 deployment on the same host fails due to a modified boot table entry. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011306[BZ#2011306])
```

Comment:
I have run into the same issue deploying back-to-back 4.9 cluster installs.

Reply:
Thanks @imiller0 - I've updated the known issue slightly to reflect this.